copytypes.js

```diff
@@ -0,0 +1,80 @@
+const fs = require("fs").promises;
+const path = require("path");
+const { exec } = require("child_process");
+
+const destinationFolder = "frontend/types";
+
+// Adjust these paths as per your project structure
+const files = [
+  {
+    src: path.join(__dirname, "./backend/types/generated/contentTypes.d.ts"),
```

Comment:
We should probably use an os agnostic package, while none of us use windows there might be some external users or fork/clone this.

Reply:
True I'll make it more agnostic but that also means other dependency for package.json

Reply:
Added agonistc way.

Reply:
Resolved made it agnostic.